src/less/main.less

```diff
@@ -2496,3 +2497,71 @@ div.shake-alert {
     height: 0;
   }
 }
+
+// Billing
+.billing {
```

Comment:
The right panel is a bit narrow on smaller screens, could the left panel be reduced in width a little to make room on smaller screens? See screenshot at https://scrn.li/5g9mdd6VkTrsHJ
We could probably reduce the left menu width too to make more space ( by adjusting .leftFixed and .rightFluid)

Reply:
I reduced the width, but only in this class, .rightFluid was a bit too prominent and changes there broke some different things...